packages/phrases/src/locales/zh-cn/translation/admin-console/errors.ts

```diff
@@ -6,6 +6,7 @@ const errors = {
   missing_total_number: '无法从返回的头部信息中找到 Total-Number',
   invalid_uri_format: '无效的 URI 格式',
   invalid_origin_format: '无效的 URI origin 格式',
+  invalid_error_message_format: '非法的错误信息格式',
   required_field_missing: '请输入{{field}}',
   required_field_missing_plural: '至少需要输入一个{{field}}',
```

Comment:
nit: just to double check, which language will the values of {{field}} be? if it's not in Chinese, maybe we should add space before the {{field}} placeholder?

Reply:
These fields are form field labels, and I believe most of them have been translated into Chinese as well. How about we revisit this one if we observe any issue in the admin console?

Reply:
yep, I believe somebody has already reviewed this before launch.
leave it alone 😄

Reply:
Cool!